backend/app/core/daily_journal_pdf.py

```
@@ -0,0 +1,304 @@
+        pdf.set_font("malgun", "B", 7.5)
+        pdf.cell(col_w[0], row_h, f" {label}", border=1, fill=True)
+        pdf.set_font("malgun", "", 7.5)
+        pdf.cell(col_w[1], row_h, f" {field}", border=1, fill=True)
+        pdf.cell(col_w[2], row_h, f" {product}", border=1, fill=True)
+        pdf.cell(col_w[3], row_h, f" {amount or '-'}", border=1, fill=True)
+        cy += row_h
+
+    return cy - y
+
+
+def _render_dj_on_current_page(pdf: JournalPDF, dj: DailyJournal):
```

Ruff (0.15.10)

[warning] Missing return type annotation for private function `_render_dj_on_current_page`

Add return type annotation: `None`

(ANN202)